roles/openshift_prometheus/meta/main.yaml

```diff
@@ -1,5 +1,5 @@
 galaxy_info:
-  author: OpenShift Development <dev@lists.openshift.redhat.com>
+  author: Zohar Galor
```

Comment:
Wouldn't you prefer this ref the dev list so you are not the sole keeper to the kingdom?